aws-codegurureviewer-repositoryassociation/src/main/java/software/amazon/codegurureviewer/repositoryassociation/Translator.java

```diff
@@ -79,7 +92,13 @@ static ResourceModel translateFromReadResponse(final DescribeRepositoryAssociati
         if(!providerType.equals(ProviderType.CODE_COMMIT)) {
             resourceModelBuilder.owner(awsResponse.repositoryAssociation().owner());
         }
-
+        Map<String, String> tags = awsResponse.tags();
```

Comment:
Can tags field be null in awsResponse?

Reply:
We return `Tags` field for DescribeRepositoryAssociationResponse. This field can be empty but not null.